lib/segment/src/vector_storage/simple_vector_storage.rs

```diff
@@ -183,45 +183,8 @@ impl VectorStorage for SimpleVectorStorage {
         self.db_wrapper.flusher()
     }
 
-    fn quantize(
-        &mut self,
-        path: &Path,
-        quantization_config: &QuantizationConfig,
-        max_threads: usize,
-        stopped: &AtomicBool,
-    ) -> OperationResult<()> {
-        let vector_data_iterator = (0..self.vectors.len() as u32).map(|i| self.vectors.get(i));
-        self.quantized_vectors = Some(QuantizedVectors::create(
-            vector_data_iterator,
-            quantization_config,
-            self.distance,
-            self.dim,
-            self.vectors.len(),
-            path,
-            false,
-            max_threads,
-            stopped,
-        )?);
-        Ok(())
-    }
-
-    fn load_quantization(&mut self, path: &Path) -> OperationResult<()> {
-        if QuantizedVectors::config_exists(path) {
-            self.quantized_vectors = Some(QuantizedVectors::load(path, false, self.distance)?);
-        }
-        Ok(())
-    }
-
-    fn quantized_storage(&self) -> Option<&QuantizedVectors> {
-        self.quantized_vectors.as_ref()
-    }
-
     fn files(&self) -> Vec<std::path::PathBuf> {
```

Comment:
It is weird that the `files` implementation is now empty for `simple_vector_storage`.

The `simple_vector_storage` is persisted via rocksDB and should be part of the snapshots.

Is this `files` function limited to mmap files?

Reply:
I agree with you. I find separate RockDB files copying weird. But it's out of scope here. We may add some TODO

Reply:
I can keep track of it, thanks :+1: